.github/workflows/test.yaml

```
@@ -14,6 +14,7 @@ jobs:
   test:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

What was wrong with fail fast?

---

When fail-fast was set to true first failed action cancelled all the others.